elifesciences/builder
# ELPP-3633 Run `vagrant up` using a container for the Python part

`vagrant up` runs on the host, but delegates reading project data to `.project.py`, which we can execute inside a container. The image derived from `python:2.7` allows Mac users not to install Python for this use case.

## To test

1. Install `docker` if not already installed
2. Install `vagrant` (and `VirtualBox`) if not already installed (can't easily run in Docker, I think)
3. Run `vagrant up`
4. Choose `schematron-validator` which was our candidate example
5. See the whole provisioning succeed
